packages/ui/src/components/va-parallax/VaParallax.vue

```diff
@@ -55,19 +58,24 @@ export default defineComponent({
       top: props.reversed ? 0 : 'auto',
     }))
 
-    const rootElement: Ref<HTMLElement | null> = ref(null)
     const targetElement = computed(() => {
-      if (typeof props.target !== 'string') {
+      if (!props.target) {
         return getScrollableParent(rootElement.value?.parentElement)
       }
 
+      if (props.target instanceof HTMLElement) { // there is a bug if to target passed ref
+        return props.target
+      }
+
       const element = document.querySelector(props.target)
 
       if (element) { return element }
 
-      throw new Error('VaParallax target prop got wrong selector. Target is null')
+      console.warn('VaParallax target prop got wrong selector. Target is null')
```

Comment:
Warnings should be only in dev.

Reply:
Maybe we can me VaWarn method here?:
vuestic-ui/packages/ui/src/utils/global-utils.ts